lib/rucio/tests/test_clients.py

```diff
@@ -58,10 +58,11 @@ def testUserpassNoCACert(self):
         creds = {'username': 'wrong', 'password': 'secret'}
         BaseClient(account='root', auth_type='userpass', creds=creds)
 
-    def testx509(self):
-        """ CLIENTS (BASECLIENT): authenticate with x509."""
-        creds = {'client_cert': self.usercert}
-        BaseClient(account='root', ca_cert=self.cacert, auth_type='x509', creds=creds)
+#
+#    def testx509(self):
```

Comment:
Why is this test disabled?

Reply:
it fail with some openssl error for all python versions. I think it's related to urlib3. I forgot to open a separate issue.

Reply:
Internally to the docker yes. With the server running in a container and the client outside it, it gives some ssl errors.

Reply:
I got it. it was some wrong parameters in the configuration. committing...